<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::PrivateActivityManager</class>
 <widget class="QWidget" name="tt3::gui::PrivateActivityManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="filterWidget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="privateActivitiesTreeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="buttonsWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="4" column="0">
       <widget class="QPushButton" name="startPrivateActivityPushButton">
        <property name="text">
         <string>Start private activity</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/StartPublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/StartPublicActivitySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line_1">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="destroyPrivateActivityPushButton">
        <property name="text">
         <string>Destroy private activity</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/DestroyPublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/DestroyPublicActivitySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="createPrivateActivityPushButton">
        <property name="text">
         <string>Create private activity</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/CreatePublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/CreatePublicActivitySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="stopPrivateActivityPushButton">
        <property name="text">
         <string>Stop private activity</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/StopPublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/StopPublicActivitySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="modifyPrivateActivityPushButton">
        <property name="text">
         <string>Modify private activity</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/ModifyPublicActivitySmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/ModifyPublicActivitySmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>privateActivitiesTreeWidget</tabstop>
  <tabstop>modifyPrivateActivityPushButton</tabstop>
  <tabstop>createPrivateActivityPushButton</tabstop>
  <tabstop>destroyPrivateActivityPushButton</tabstop>
  <tabstop>startPrivateActivityPushButton</tabstop>
  <tabstop>stopPrivateActivityPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>privateActivitiesTreeWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>tt3::gui::PrivateActivityManager</receiver>
   <slot>_privateActivitiesTreeWidgetCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>privateActivitiesTreeWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>tt3::gui::PrivateActivityManager</receiver>
   <slot>_privateActivitiesTreeWidgetCustomContextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createPrivateActivityPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::PrivateActivityManager</receiver>
   <slot>_createPrivateActivityPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifyPrivateActivityPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::PrivateActivityManager</receiver>
   <slot>_modifyPrivateActivityPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destroyPrivateActivityPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::PrivateActivityManager</receiver>
   <slot>_destroyPrivateActivityPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startPrivateActivityPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::PrivateActivityManager</receiver>
   <slot>_startPrivateActivityPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopPrivateActivityPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::PrivateActivityManager</receiver>
   <slot>_stopPrivateActivityPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::gui::PrivateActivityManager</receiver>
   <slot>_filterLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_privateActivitiesTreeWidgetCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>_privateActivitiesTreeWidgetCustomContextMenuRequested(QPoint)</slot>
  <slot>_createPrivateActivityPushButtonClicked()</slot>
  <slot>_modifyPrivateActivityPushButtonClicked()</slot>
  <slot>_destroyPrivateActivityPushButtonClicked()</slot>
  <slot>_startPrivateActivityPushButtonClicked()</slot>
  <slot>_stopPrivateActivityPushButtonClicked()</slot>
  <slot>_filterLineEditTextChanged(QString)</slot>
 </slots>
</ui>
